content/en/docs/Advanced/build.md

```diff
@@ -122,12 +122,12 @@ $ curl http://<node-ip>:$PORT/hello-kairos.iso -o output.iso
 
 It is possible to use the CRD to prepare artifacts required for netbooting, by enabling `netboot: true` for instance:
 
-```yaml
+```yaml {class="meta-distro"}
```

Comment:
is it possible to override this template via the base templates somehow to always append the this class? Otherwise this will only works if we remember to add the class manually no? 

IIRC this ` ```LANGUAGE` stuff is provided via a template or function so we may be able to extend it directly in there?

Reply:
yeah good point, let me see what I can find

Reply:
refactored